lib/bolt/node/ssh.rb

```diff
@@ -54,9 +54,19 @@ def disconnect
       end
     end
 
-    def execute(command, options = {})
+    def execute(command, sudoable: false, **options)
       result_output = Bolt::Node::ResultOutput.new
       status = {}
+      use_sudo = sudoable && (@sudo || @run_as)
+      sudo_prompt = '[sudo] Bolt needs to run as another user, password: '
+      if use_sudo
+        user_clause = if @run_as
+                        "-u #{@run_as}"
+                      else
+                        ''
+                      end
+        command = "sudo -S #{user_clause} -p '#{sudo_prompt}' #{command}"
```

Comment:
Couldn't this use `"#{@sudo} ...` to make it slightly easier to add different sudo options in the future?

Reply:
I guess I should just look at doing that, doesn't seem too hard to add a few.

Reply:
Almost every part needs to be parameterized to add another method so it didn't seem worth it to do it until we actually added one.